mobile/screens/HomeScreen.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+
+import { View, Button, FlatList, Text, StyleSheet }
+  from 'react-native'
+
+export class HomeScreen extends Component {
+  static navigationOptions = ({navigation}) => ({
+    title: 'Your shared buylist',
+  })
+  render() {
+    const { navigate, goBack } = this.props.navigation
+    const username = this.props.navigation.state.username
+    const password = this.props.navigation.state.password
+    return (
+      <View style={styles.container}>
+        <Text style={styles.h1}>Welcome back!</Text>
+      </View>
+    )
+  }
+}
+
+const styles = StyleSheet.create({
```

Comment:
I see that you used most of this stylesheet in the Login as well, wouldn't it be good to extract it in a separate file and decouple it from the structural code?

Reply:
I think you are very right. I'm not sure exactly how to do that yet, but we'll do that in future pull requests.

Reply:
Ok, sounds good to me.